src/main/tut/README.md

```diff
@@ -146,6 +146,16 @@ logging.
   .attemptRun)
 ```
 
+#### using
+Ensure that a resource is "closed" when a task completes, regardless of whether it's successful.
```

Comment:
Might want to mention here that it requires a CanClose trait, and that Closeable and ExecutorService have predefined ones.